### DISCUSSION TITLE
UI crashes trying to access settings - submitted bug report thru app

### DISCUSSION THREAD

**Discussion by mmiller7** [meshtastic/Discussions#155] [Android]
Attempted to submit a bug report thru the app, it says to post here to match it up.

I've had a significant number of times across 2 different brand/model nodes where I try and make an adjustment and when I go to Radio Settings > Tap a section (e.g. LoRA) it gets part way thru the loading bar and just hangs.  Nomater how long I wait it never gets to the end.  If I cancel/close and go back it hangs at 0%.  Only solution is I have to force-kill the Meshtastic app thru Android app manager and then re-open it, let it re-connect to the node, and finally go back into settings usually works fine.

Not sure what other info is needed from me.

**Comment by garthvh** [comment]
The MUI uses the same connection as the app, they can't be used at the same time.

**Reply by mmiller7** [reply]
> The MUI uses the same connection as the app, they can't be used at the same time.

What is "MUI"?  I was not using anything except the Android app, I don't know what could have been using something "at the same time"?

**Reply by mmiller7** [reply]
How is "MUI" turned off so I can use the app reliably?  This is still an issue, I am still only using the Android app to control it.

I don't see any procedures for that, or definitions of "MUI" searching any of the Meshtastic documentation I can find.
Searching "Discussions" this is the only hit for the term "MUI".

**Reply by madeofstown** [reply]
> How is "MUI" turned off so I can use the app reliably? This is still an issue, I am still only using the Android app to control it.
> 
> I don't see any procedures for that, or definitions of "MUI" searching any of the Meshtastic documentation I can find. Searching "Discussions" this is the only hit for the term "MUI".

Unless you are running the unreleased UI reworked firmware this isn't your issue. The previous commenter made some assumptions on I'm not sure what. 
What version of the app are you using and are you denying it any permissions that it requested?

**Reply by mmiller7** [reply]
> > How is "MUI" turned off so I can use the app reliably? This is still an issue, I am still only using the Android app to control it.
> > I don't see any procedures for that, or definitions of "MUI" searching any of the Meshtastic documentation I can find. Searching "Discussions" this is the only hit for the term "MUI".
> 
> Unless you are running the unreleased UI reworked firmware this isn't your issue. The previous commenter made some assumptions on I'm not sure what. What version of the app are you using and are you denying it any permissions that it requested?

I am not aware of any unreleased UIs for either the T-Beam or Station G2 nodes (both of which have done this crashing thing with official firmware 2.5.15 from web-flasher).

Android app from the Google Play store, if I'm understanding correctly the app is also reporting its version as 2.5.15.  I have experienced this on an Android 9 Samsung tablet and an Android 14 Samsung S23 phone.

I don't know if its related or not but I also have observed times where the node is disconnected (e.g. I unplugged it to try moving so it can't be connected) and the Android app still thinks its connected and reports connected.

When it happens, I can force-kill the Android app and immediately reopen it, then get it to reconnect to the node and everything works as expected again.

To my knowledge I have not denied any permissions to the app, in settings it says "camera" is not allowed but it hasn't asked for camera yet.  It has location, nearby devices, and notifications as allowed.
